src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { ErorTypes } from '../types/ErrorTypes';
+
+type Props = {
+  isErrorMessage: ErorTypes,
+  setIsErrorMessage: (value: ErorTypes) => void,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  isErrorMessage, setIsErrorMessage,
+}) => {
+  const [onCloseError, setOnCloseError] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setIsErrorMessage(ErorTypes.none);
+    }, 3000);
+  }, [isErrorMessage]);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        { hidden: onCloseError || isErrorMessage === ErorTypes.none },
+      )}
+    >
+      <button
+        aria-label="delete"
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={() => setOnCloseError(true)}
+      />
+      {isErrorMessage === ErorTypes.load && ('Unable to load a todo')}
+      {isErrorMessage === ErorTypes.upload && ('Unable to add a todo')}
+      {isErrorMessage === ErorTypes.delete && ('Unable to delete a todo')}
+      {isErrorMessage === ErorTypes.title && ('Title can\'t be empty')}
```

Comment:
it can be simplify: 

```suggestion
      {ERROR_TEXT[isErrorMessage] || ''}
```
and in your constants 
const ERROR_TEXT = {
          [ErorTypes.load]: 'Unable to load a todo',
          [ErorTypes.upload]: 'Unable to add a todo',
          [ErorTypes.delete]: 'Unable to delete a todo',
          [ErorTypes.title]: 'Title can\'t be empty',
    }

Reply:
linter says in ERROR_TEXT must be one more parameter [ErorTypes.none] :)